server/ladders-remote.ts

```diff
@@ -149,4 +140,37 @@ export class LadderStore {
 	static async visualizeAll(username: string) {
 		return [`<tr><td><strong>Please use the official client at play.pokemonshowdown.com</strong></td></tr>`];
 	}
+
+	/**
+	 * Calculates Elo for quick display, matching the formula on loginserver
+	 */
+	// see lib/ntbb-ladder.lib.php in the pokemon-showdown-client repo for the login server implementation
+	// *intentionally* different from calculation in ladders-local, due to the high activity on the main server
```

Comment:
Wait, in hindsight I think the only difference is because of ladder decay, which isn't implemented here.

Is the only difference here the `1300` vs the `1350`? Maybe we *should* merge them.

Reply:
That's not the only difference; the numbers for scaling when elo < 1200 are also different.
ladders-local:
```
if (score < 0.5) {
  K = 10 + (previousUserElo - 1000) * 40 / 200;
} else if (score > 0.5) {
  K = 90 - (previousUserElo - 1000) * 40 / 200;
}
```

ladders-remote:
```
if (score < 0.5) {
  K = 20 + (previousUserElo - 1000) * 30 / 100;
} else if (score > 0.5) {
  K = 80 - (previousUserElo - 1000) * 30 / 100;
}
```

ladders-remote uses 20/80 instead of 10/90, and multiplies by 30 instead of 40.

Reply:
Interesting! We probably want to investigate the formulas more, later. But that's probably a job for a different PR.